js/about/history.js

```diff
@@ -71,7 +72,11 @@ class GroupedHistoryList extends ImmutableComponent {
       }
       return result
     })
-    if (reduced) return reduced
+    if (reduced) {
+      return Array.isArray(reduced)
+        ? reduced
+        : [{date: this.getDayString(locale, reduced), entries: [reduced]}]
```

Comment:
This line has the fix for showing single entries. Before, it would just return the object (which fails when trying to do the map() in the render for the HistoryDay control)